src/main/resources/openolitor-server-test.conf

```diff
@@ -57,7 +57,7 @@ openolitor {
     }
 
     converttopdf {
-      endpoint = "http://pdftools.openolitor.ch/convert2pdf"
+      endpoint = "http://pdftools.openolitor.ch/lool/convert-to/pdf"
```

Comment:
I am not familiar with the setup but `server-test` does not look like local to me

Reply:
yes, sorry. The branch name considered originally only the local configuration. The server test configuration is only one example, we are currently not using this file for deploying test. All *.conf files are in the deployment repository as some of the data is sensitive.